admin-dev/themes/new-theme/js/pages/product/edit/combinations-manager.ts

```diff
@@ -419,43 +516,96 @@ export default class CombinationsManager {
   }
 
   /**
-   * @param {HTMLElement} checkedInput
+   * @param {HTMLElement} input of the same table row
+   *
+   * @returns {Number}
    *
    * @private
    */
-  private async updateDefaultCombination(checkedInput: HTMLInputElement): Promise<void> {
-    const checkedInputs = this.$combinationsContainer.find(
-      `${CombinationsMap.isDefaultInputsSelector}:checked`,
-    );
-    const checkedDefaultId = this.findCombinationId(checkedInput);
+  private findCombinationId(input: HTMLElement): any {
```

Comment:
I guess `.val()` returns something? We are probably able to avoid the any type there